nipype/pipeline/engine/utils.py

```diff
@@ -1314,18 +1314,17 @@ def export_graph(graph_in,
 def format_dot(dotfilename, format='png'):
     """Dump a directed graph (Linux only; install via `brew` on OSX)"""
     if format != 'dot':
-        cmd = 'dot -T%s -O \'%s\'' % (format, dotfilename)
+        dot_base =  os.path.splitext(dotfilename)[0]
+        formatted_dot = '{}.{}'.format(dot_base, format)
+        cmd = 'dot -T{} -o"{}" "{}"'.format(format, formatted_dot, dotfilename)
```

Comment:
I think this needs to be uppercase O

Reply:
nope, `-ofile` is what we should be using.